crates/holochain_sqlite/tests/loom.rs

```diff
@@ -1,17 +1,58 @@
-#[cfg(loom)]
-mod tests {
-    use loom::sync::atomic::AtomicUsize;
+#[cfg(feature = "test_utils")]
+mod common;
 
-    use std::sync::atomic::Ordering::SeqCst;
-    use std::sync::Arc;
+#[cfg(all(loom, feature = "test_utils"))]
+mod tests {
+    use crate::common::TestDatabaseKind;
+    use holochain_sqlite::db::DbWrite;
+    use holochain_sqlite::error::DatabaseResult;
+    use std::time::Duration;
 
     #[test]
-    fn test_concurrent_logic() {
+    fn multiple_readers() {
         loom::model(|| {
-            let v1 = Arc::new(AtomicUsize::new(0));
-            let v2 = v1.clone();
+            let tmp_dir = tempfile::TempDir::new().unwrap();
+            let db_handle = DbWrite::open(&tmp_dir.into_path(), TestDatabaseKind::new()).unwrap();
+
+            let rt = tokio::runtime::Builder::new_current_thread()
```

Comment:
This is important, we don't want to go launching a thread pool that loom doesn't control and it complains at runtime if you do. The Tokio tests also seem to use this approach.